From c6f69143bb5a6c95f812f9d71d7a78f6b2c2eaeb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Thu, 18 Sep 2014 12:21:12 +0200
Subject: [PATCH] Do not show duplicated menu items

When the support to use numpad accelerators for zoom-{in,out,reset}
was added (3a168815), by mistake, we have added duplicated buttons
in View -> Zoom.

https://bugzilla.redhat.com/show_bug.cgi?id=883433
(cherry picked from commit 55cdb9867df05f1c4f6c8e569a6f0c1e0bc28d99)
---
 src/virt-viewer.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/virt-viewer.xml b/src/virt-viewer.xml
index 760550e..6f2eebb 100644
--- a/src/virt-viewer.xml
+++ b/src/virt-viewer.xml
@@ -141,7 +141,7 @@
                               <object class="GtkImageMenuItem" id="menu-view-kp-zoom-in">
                                 <property name="accel_path">&lt;virt-viewer&gt;/view/kp-zoom-in</property>
                                 <property name="label">gtk-zoom-in</property>
-                                <property name="visible">True</property>
+                                <property name="visible">False</property>
                                 <property name="can_focus">False</property>
                                 <property name="use_action_appearance">False</property>
                                 <property name="use_underline">True</property>
@@ -165,7 +165,7 @@
                               <object class="GtkImageMenuItem" id="menu-view-kp-zoom-out">
                                 <property name="accel_path">&lt;virt-viewer&gt;/view/kp-zoom-out</property>
                                 <property name="label">gtk-zoom-out</property>
-                                <property name="visible">True</property>
+                                <property name="visible">False</property>
                                 <property name="can_focus">False</property>
                                 <property name="use_action_appearance">False</property>
                                 <property name="use_underline">True</property>
@@ -195,7 +195,7 @@
                               <object class="GtkImageMenuItem" id="menu-view-kp-zoom-reset">
                                 <property name="accel_path">&lt;virt-viewer&gt;/view/kp-zoom-reset</property>
                                 <property name="label">gtk-zoom-100</property>
-                                <property name="visible">True</property>
+                                <property name="visible">False</property>
                                 <property name="can_focus">False</property>
                                 <property name="use_action_appearance">False</property>
                                 <property name="use_underline">True</property>
